core/utils/str.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace Elementor\Core\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Str {
```

Comment:
```suggestion
class String {
```

Reply:
I'm not sure that's a good idea to use a variable type as class name. 
Plus, `Str` is the class name that Laravel uses for their string util so me and @Nevoss thought that's a better fit.